OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1796,6 +1796,27 @@ internal static void UpgradeWeaponRules(ModData modData, int engineVersion, ref
 					}
 				}
 
+				if (engineVersion < 20180120)
+				{
+					if (node.Key.StartsWith("Warhead", StringComparison.Ordinal) && node.Value.Value == "CreateEffect")
+					{
+						var victimScanRadius = node.Value.Nodes.FirstOrDefault(n => n.Key == "VictimScanRadius");
+						if (victimScanRadius != null)
+						{
+							if (FieldLoader.GetValue<int>(victimScanRadius.Key, victimScanRadius.Value.Value) == 0)
+							{
+								node.Value.Nodes.Add(new MiniYamlNode("ImpactActors", "false"));
+							}
+							node.Value.Nodes.Remove(victimScanRadius);
```

Comment:
> OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs:L1809: [SA1513] Statements or elements wrapped in curly brackets must be followed by a blank line.

Reply:
Single line if statements should not have curly brackets in the first place.

Reply:
Ok, but for readability ther should be an empty line between the .Add and the .Remove either way.

Reply:
yes :)